ports/geckolib/gecko_bindings/bindings.rs

```diff
@@ -276,7 +276,7 @@ extern "C" {
     pub fn Servo_StyleSheetHasRules(sheet: *mut RawServoStyleSheet) -> bool;
     pub fn Servo_InitStyleSet() -> *mut RawServoStyleSet;
     pub fn Servo_DropStyleSet(set: *mut RawServoStyleSet);
-    pub fn Servo_ParseStyleAttribute(bytes: *const u8, length: u32,
```

Comment:
Yes and no. https://bugzilla.mozilla.org/show_bug.cgi?id=1287383 hasn't landed yet, and it wasn't cherry-picked onto the stylo branch when Emilio did the rebase. When that patch lands, I'll do a resync.

We want it such that the code in https://github.com/servo/gecko-dev (stylo branch) always matches the code on Servo master. Generally if a patch hasn't yet landed in Gecko (but the corresponding patch has landed in Servo), you should just cherry-pick onto the fork (which wasn't done here, or it was lost in the rebase).

I could cherry pick, however the Gecko patch should land soon anyway, after which I can do another sync.


cc @heycam